mxcube3/ui/actions/queue.js

```diff
@@ -74,13 +74,11 @@ export function addSamplesToQueue(sampleDataList) {
   return function (dispatch) {
     dispatch(queueLoading(true));
 
-    dispatch(addSamplesToQueueAction(sampleDataList));
-
     sendAddQueueItem(sampleDataList).then((response) => {
       if (response.status >= 400) {
         dispatch(showErrorPanel(true, 'Server refused to add sample'));
-        const sampleIDList = sampleDataList.map((sampleData) => (sampleData.sampleID));
```

Comment:
@meguiraun 
Small question: before samples were added to the queue (`addSamplesToQueueAction`) - now with the new code, I don't see where they are added ; could you please explain to me how it works, just to keep me in sync.